ar_exercise/exercise.rb

```diff
@@ -51,7 +51,20 @@ class Apartment < ActiveRecord::Base
 # Hint, the following methods will help: `new`, `create`, `save`, `uddate`, `destroy`
 
 # Create 3 new apartments, and save them to the DB
+new_apartment1 = Apartment.create (address:1000, monthly_rent:$3000, sqft: 1000, num_beds:27, num_baths:2 )
+new_apartment2 = Apartment.create (address:1800, monthly_rent:$2000, sqft: 1000, num_beds:27, num_baths:2 )
+new_apartment3 = Apartment.create (address:1500, monthly_rent:$1000, sqft: 600, num_beds:1, num_baths:0 )
 # Create at least 9 new tenants and save them to the DB. (Make sure they belong to an apartment)
+Joe = Tenant.create (name:"Joe", age: 12, gender:"Male", apartment_id:324)
+Tom = Tenant.create (name:"Joe", age: 19, gender:"Male", apartment_id:326)
+Dick = Tenant.create (name:"Joe", age: 19, gender:"Male", apartment_id:328)
+Harry = Tenant.create (name:"Joe", age: 22, gender:"Male", apartment_id:724)
+Suzy = Tenant.create (name:"Joe", age: 23, gender:"Female", apartment_id:524)
+Clyde = Tenant.create (name:"Joe", age: 25, gender:"Male", apartment_id:924)
+Beatrice = Tenant.create (name:"Joe", age: 27, gender:"Female", apartment_id:320)
+Claudette = Tenant.create (name:"Joe", age: 40, gender:"Female", apartment_id:424)
+Carmen = Tenant.create (name:"Joe", age: 30, gender:"Female", apartment_id:824)
+Naomi = Tenant.create (name:"Joe", age: 55, gender:"Female", apartment_id:800)
```

Comment:
You've hard coded the apartment_id, which generally a bad practice. Fortunately ActiveRecord is super smart and will add the appropriate apartment_id when we create a tenant on an apartment
```rb
new_apartment1.tenants.create("a bunch of key value pairs, not including apartment_id")
```